mslib/msui/mss_pyui.py

```diff
@@ -894,8 +894,11 @@ def main():
     logging.info("Python Version: %s", sys.version)
     logging.info("Platform: %s (%s)", platform.platform(), platform.architecture())
 
-    read_config_file()
-    logging.info("Launching user interface...")
+    try:
+        read_config_file()
+    except FileNotFoundError as ex:
```

Comment:
Now that the settings file is being created, does this error ever occur with the default parameter for read_config_file?

Reply:
Agree, it's "double fixed" now. And the fs/non-fs code path is different.

Reply:
interesting, we have  two tests which don't fail. So yes that can be reached.

Reply:
It still can happen, but I guess rather seldom.  One case is you set a path where you don't have create rights. 

decided to exit for that case and throw an error message.